FAILED TEST LOG
  ✘ You should run your script by executing it

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(lastCommand[0] === './fortune.sh' && /\/project$/.test(cwd))

at Context.<anonymous> (test/1270.test.js:9:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+('|"|)hello\1[ \t]+==[ \t]+('|"|)hello\2[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/gm.test(lastCommand))

at Context.<anonymous> (test/1290.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+('|")hello world\1[ \t]+=~[ \t]+\^h[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1330.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+('|")hello world\1[ \t]+=~[ \t]+\^h[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1330.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*echo[ \t]+('|"|)\$VAR\1[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1360.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+("|)\$VAR\1[ \t]+==[ \t]+('|")hello world\2[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1370.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+("|)\$VAR\1[ \t]+==[ \t]+('|")hello world\2[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1370.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+("|)\$VAR\1[ \t]+==[ \t]+('|")hello world\2[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1370.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+("|)\$VAR\1[ \t]+==[ \t]+('|")hello world\2[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1370.test.js:8:5)
FAILED TEST LOG
  ✘ You should enter the suggested command in the terminal

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*\[\[[ \t]+("|)\$VAR\1[ \t]+=~[ \t]+\\\?\$[ \t]+\]\][ \t]*;[ \t]*echo[ \t]+\$\?[ \t]*;?\s*$/.test(lastCommand))

at Context.<anonymous> (test/1380.test.js:8:5)
FAILED TEST LOG
  ✘ Your script should have the correct condition for your "until" loop

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: conditional binary operator expected
./fortune.sh: line 15: syntax error near `=?'
./fortune.sh: line 15: `until [[ $QUESTION =? ]]'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should have the correct condition for your "until" loop

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*until[ \t]+\[\[[ \t]+\$QUESTION[ \t]+=~[ \t]+\\\?\$[ \t]+\]\]/gm.test(scriptFile) && output.match(/Ask a yes or no question:/gm).length === 2)

at Context.<anonymous> (test/1390.test.js:9:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggested "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^\s*if[ \t]+\[\[[ \t]+![ \t]+\$1[ \t]+\]\]/gm.test(scriptFile) && /Ask a yes or no question:/g.test(output))

at Context.<anonymous> (test/1410.test.js:9:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 14: syntax error near unexpected token `}'
./fortune.sh: line 14: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 15: syntax error near unexpected token `}'
./fortune.sh: line 15: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: syntax error near unexpected token `}'
./fortune.sh: line 15: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 16: syntax error near unexpected token `}'
./fortune.sh: line 16: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 16: syntax error near unexpected token `}'
./fortune.sh: line 16: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should have the suggested "if" statement added correctly

Error: write EPIPE
at afterWriteDispatched (node:internal/stream_base_commons:160:15)
at writeGeneric (node:internal/stream_base_commons:151:3)
at Socket._writeGeneric (node:net:962:11)
at Socket._write (node:net:974:8)
at writeOrBuffer (node:internal/streams/writable:392:12)
at _write (node:internal/streams/writable:333:10)
at Writable.write (node:internal/streams/writable:337:10)
at Socket.<anonymous> (test/utils.js:116:19)
at Socket.emit (node:events:529:35)
at addChunk (node:internal/streams/readable:368:12)
at readableAddChunk (node:internal/streams/readable:337:11)
at Readable.push (node:internal/streams/readable:278:10)
at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)

An error occurred running ./fortune.sh
./fortune.sh: line 16: syntax error near unexpected token `}'
./fortune.sh: line 16: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: done() called multiple times in test <Your script should have the suggested "if" statement added correctly> of file /workspace/project/.freeCodeCamp/test/1410.test.js; in addition, done() received error: Error: Command failed: ./fortune.sh
./fortune.sh: line 16: syntax error near unexpected token `}'
./fortune.sh: line 16: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12) {
code: 2,
killed: false,
signal: null,
cmd: './fortune.sh',
stdout: '\n~~ Fortune Teller ~~\n\n',
stderr: "./fortune.sh: line 16: syntax error near unexpected token `}'\n" +
"./fortune.sh: line 16: `}'\n"
}
FAILED TEST LOG

./fortune.sh: line 15: syntax error near unexpected token `}'
./fortune.sh: line 15: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: syntax error near unexpected token `}'
./fortune.sh: line 15: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should have the suggested "if" statement added correctly

Error: write EPIPE
at afterWriteDispatched (node:internal/stream_base_commons:160:15)
at writeGeneric (node:internal/stream_base_commons:151:3)
at Socket._writeGeneric (node:net:962:11)
at Socket._write (node:net:974:8)
at writeOrBuffer (node:internal/streams/writable:392:12)
at _write (node:internal/streams/writable:333:10)
at Writable.write (node:internal/streams/writable:337:10)
at Socket.<anonymous> (test/utils.js:116:19)
at Socket.emit (node:events:529:35)
at addChunk (node:internal/streams/readable:368:12)
at readableAddChunk (node:internal/streams/readable:337:11)
at Readable.push (node:internal/streams/readable:278:10)
at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)

An error occurred running ./fortune.sh
./fortune.sh: line 17: syntax error near unexpected token `}'
./fortune.sh: line 17: `}'
  ✘ Your script should have the suggested "if" statement added correctly

Error: done() called multiple times in test <Your script should have the suggested "if" statement added correctly> of file /workspace/project/.freeCodeCamp/test/1410.test.js; in addition, done() received error: Error: Command failed: ./fortune.sh
./fortune.sh: line 17: syntax error near unexpected token `}'
./fortune.sh: line 17: `}'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5) {
code: 2,
killed: false,
signal: null,
cmd: './fortune.sh',
stdout: '\n~~ Fortune Teller ~~\n\n',
stderr: "./fortune.sh: line 17: syntax error near unexpected token `}'\n" +
"./fortune.sh: line 17: `}'\n"
}
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 15: syntax error near unexpected token `fi'
./fortune.sh: line 15: `  fi'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: syntax error near unexpected token `fi'
./fortune.sh: line 15: `  fi'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 15: syntax error near unexpected token `fi'
./fortune.sh: line 15: `  fi'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: syntax error near unexpected token `fi'
./fortune.sh: line 15: `  fi'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 15: syntax error near unexpected token `fi'
./fortune.sh: line 15: `  fi'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: syntax error near unexpected token `fi'
./fortune.sh: line 15: `  fi'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 14: syntax error near unexpected token `fi'
./fortune.sh: line 14: `  fi'
  ✘ Your script should have the suggested "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 14: syntax error near unexpected token `fi'
./fortune.sh: line 14: `  fi'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should have the suggests "if/else" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/Try again\. Make sure it ends with a question mark:/g.test(output) && /Ask a yes or no question:/g.test(output))

at Context.<anonymous> (test/1412.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./.freeCodeCamp/test.sh: line 15: syntax error near unexpected token `fi'
./.freeCodeCamp/test.sh: line 15: `  fi'
  ✘ Your script should have the suggests "if/else" statement added correctly

Error: Command failed: ./.freeCodeCamp/test.sh
./.freeCodeCamp/test.sh: line 15: syntax error near unexpected token `fi'
./.freeCodeCamp/test.sh: line 15: `  fi'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./.freeCodeCamp/test.sh: line 16: syntax error near unexpected token `fi'
./.freeCodeCamp/test.sh: line 16: `  fi'
  ✘ Your script should have the suggests "if/else" statement added correctly

Error: Command failed: ./.freeCodeCamp/test.sh
./.freeCodeCamp/test.sh: line 16: syntax error near unexpected token `fi'
./.freeCodeCamp/test.sh: line 16: `  fi'

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should call the suggested function correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/Try again\. Make sure it ends with a question mark:/g.test(output) && /GET_FORTUNE[ \t]+again[ \t]*;?/g.test(scriptFile))

at Context.<anonymous> (test/1413.test.js:9:5)
FAILED TEST LOG
  ✘ Your script should call the suggested function correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/Try again\. Make sure it ends with a question mark:/g.test(output) && /GET_FORTUNE[ \t]+again[ \t]*;?/g.test(scriptFile))

at Context.<anonymous> (test/1413.test.js:9:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should call your function correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/Ask a yes or no question:\s*Try again\. Make sure it ends with a question mark:/g.test(output) && scriptFile.match(/GET_FORTUNE/gm).length > 2)

at Context.<anonymous> (test/1416.test.js:9:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG

./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG

./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 15: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at Socket.<anonymous> (node:internal/child_process:450:11)
at Socket.emit (node:events:517:28)
at Pipe.<anonymous> (node:net:350:12)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggests "if" statement added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/\n{2}(Yes|No|Maybe|Outlook good|Don't count on it|Ask again later)\n$/g.test(output))

at Context.<anonymous> (test/1425.test.js:8:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG

./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file
  ✘ Your script should have the suggests "if" statement added correctly

Error: Command failed: ./fortune.sh
./fortune.sh: line 27: unexpected EOF while looking for matching `"'
./fortune.sh: line 28: syntax error: unexpected end of file

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG
  ✘ Your script should have the suggested comment added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(/^#[ \t]*Program to run my other four programs[ \t]*$/gm.test(scriptFile))

at Context.<anonymous> (test/1460.test.js:8:5)
FAILED TEST LOG
  ✘ You should run your script by executing it

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(lastCommand[0] === './five.sh' && /\/project$/.test(cwd))

at Context.<anonymous> (test/1480.test.js:9:5)
FAILED TEST LOG
  ✘ Your script should have the suggested commands added correctly

Error: Command failed: ./.freeCodeCamp/test.sh
./.freeCodeCamp/test.sh: line 6: .: filename argument required
.: usage: . filename [arguments]

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG
  ✘ Your script should have the suggested commands added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(test1 && test2 && test3 && test4 && /\.\/countdown\.sh[ \t]+3/g.test(scriptFile))

at Context.<anonymous> (test/1490.test.js:15:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggested commands added correctly

Error: Command failed: ./.freeCodeCamp/test.sh
./.freeCodeCamp/test.sh: line 7: ./bingo.: No such file or directory

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG
  ✘ Your script should have the suggested commands added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(test1 && test2 && test3 && test4 && /\.\/countdown\.sh[ \t]+3/g.test(scriptFile))

at Context.<anonymous> (test/1490.test.js:15:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
FAILED TEST LOG
  ✘ Your script should have the suggested commands added correctly

Error: Command failed: ./.freeCodeCamp/test.sh
./.freeCodeCamp/test.sh: line 8: ./fortune.s: No such file or directory

at ChildProcess.exithandler (node:child_process:422:12)
at ChildProcess.emit (node:events:517:28)
at maybeClose (node:internal/child_process:1098:16)
at ChildProcess._handle.onexit (node:internal/child_process:303:5)
FAILED TEST LOG
  ✘ Your script should have the suggested commands added correctly

AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:

assert(test1 && test2 && test3 && test4 && /\.\/countdown\.sh[ \t]+3/g.test(scriptFile))

at Context.<anonymous> (test/1490.test.js:15:5)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
